Sandbox/src/Game/Interface/GearInterface.h

```diff
@@ -6,17 +6,36 @@
 class GearInterface : public IMenuTab
 {
 private:
+	static const int SLOT_SIZE = 40;
```

Comment:
This is good to have and should stay in this class. You can use to set the positions in mSlotPos and when initializing the icons.